### PR TITLE
role filter removed

### PR DESCRIPTION
**Pull request by evidentti**

### CODE REVIEW

File: app/directives/meetings/admeetings.directive.js [hunk 1]
```diff
@@ -73,7 +64,9 @@ angular.module('dashboard')
 
                         for (var j = 0; j < item.roleIDs.length; j++) {
                             var role = item.roleIDs[j].RoleName;
-                            var rVisible = self.rF ? (self.rF === role) : true;
+                            var roleId = item.roleIDs[j].RoleID;
+
+                            var rVisible = (roleId === 2) ? true : false;
```
Comment (tkenakka) [inline app/directives/meetings/admeetings.directive.js]: This is same as 
`var rVisible = roleId === 2;`

File: app/directives/meetings/admeetings.directive.html [hunk 2]
```diff
@@ -20,16 +18,8 @@
...
-                {{ctrl.roleTitle}}
-                <span class="caret"></span>
-            </button>
-            <ul uib-dropdown-menu role="menu" aria-labelledby="roleButton">
-                <li role="menuitem" ng-repeat="role in ctrl.roleData track by $index"><a ng-click="ctrl.setRoleFilter(role)">{{role}}</a></li>
-                <li ng-if="ctrl.rF" class="divider"></li>
-                <li ng-if="ctrl.rF" role="menuitem"><a ng-click="ctrl.setRoleFilter(null)">poista valinta</a></li>
-            </ul>
+        <div class="btn-group"">
+            <button id="roleButton" type="button" class="btn btn-success" translate>STR_ROLE</button>
```
Comment (tkenakka) [inline app/directives/meetings/admeetings.directive.html]: Dummy disabled

**Comment by evidentti** [comment]
This can be merged if LGTM

**Comment by tkenakka** [comment]
LGTM

**Pull request merged**